masterlist.yaml

```
@@ -9458,11 +9458,9 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/3484/'
         name: 'Civil War Aftermath'
-    dirty:
-      - <<: *quickClean
-        crc: 0xD6093F62
-        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 9
+    clean:
+      - crc: 0x2AEF51C4
+        util: 'SSEEdit v4.1.5f'
```

Double-checked this one, & it looks like you've cleaned this one previously. I get the checksum you got after I clean it:

```suggestion
    dirty:
      - <<: *quickClean
        crc: 0xD6093F62
        util: '[SSEEdit v4.1.5g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 9
```